mediator/med_phases_history_mod.F90

```diff
@@ -1769,7 +1769,7 @@ subroutine med_phases_history_set_timeinfo(gcomp, hclock, alarmname, &
        timediff(1) = nexttime - starttime - ringinterval
        call ESMF_TimeIntervalGet(timediff(2), d_r8=time_bnds(2), rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
-       call ESMF_TimeIntervalGet(timediff(1), d_r8=time_bnds(1), rc=rc)
+       call ESMF_TimeIntervalGet(timediff(1), starttime, d_r8=time_bnds(1), rc=rc)
```

Comment:
can you make this a named argument?
starttime=startime

Reply:
done!